n64.mk

```diff
@@ -32,14 +32,15 @@ N64_CHKSUM = $(N64_BINDIR)/chksum64
 N64_ED64ROMCONFIG = $(N64_BINDIR)/ed64romconfig
 N64_MKDFS = $(N64_BINDIR)/mkdfs
 N64_TOOL = $(N64_BINDIR)/n64tool
+N64_SYM = $(N64_BINDIR)/n64sym
 N64_AUDIOCONV = $(N64_BINDIR)/audioconv64
 
 N64_CFLAGS =  -march=vr4300 -mtune=vr4300 -I$(N64_INCLUDEDIR)
-N64_CFLAGS += -falign-functions=32 -ffunction-sections -fdata-sections
+N64_CFLAGS += -falign-functions=32 -ffunction-sections -fdata-sections -g
```

Comment:
It can be done, but we can't tie it to NDEBUG (which is a C preprocessor symbol but not a Makefile symbol). We should introduce some Makefile-level variable to control this.

Reply:
The important part I think is that the code is always considering symbols an optional part, and is ready to work without it. Obviously, symbolization can't work in that case, but you would still be able to get hexadecimal backtraces.

Reply:
Yeah we may introduce it in the future this is ok for now.